releasenotes/notes/better-rounding-in-Optimize1qGates-7627d0426344b48d.yaml

```
@@ -0,0 +1,4 @@
+---
+fixes:
+  - |
+    ```Optimize1qGates``` was missing some arity reduction opportunities because floating-point impression.
```

I'm not sure this actually needs the release note. Like is it something we need to document for end users? Not every bug fix goes in the release notes, normally only if it's user facing and/or of high importance. In this case we round a little and improve the amount the performance of optimize 1q by using u1 and u2 more which is good, but I'm not sure if it needs to be documented in the release notes because I don't know if people will really notice. But if you think it is necessary maybe we can rewrite it a bit to something like:

```suggestion
    The :class:`qiskit.transpiler.passes.Optimize1qGates` transpiler pass  was previously missing some opportunities
    for simplifying arbitrary rotation gates to more efficient versions because of unnecessarily high floating-point
    precision. This has been addressed by rounding the parameters to a fixed precision.
```

Is there a bug number associated with this issue? If there is we could add a link to the bug at the end. too